packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInstructionSetInlineSource.js

```diff
@@ -47,6 +47,11 @@ export function completeBoundaryWithStyles(
   const dependencies = [];
   let href, precedence, attr, loadingState, resourceEl, media;
 
+  function cleanupWith(cb) {
+    delete this['_p'];
```

Comment:
This should probably be `this._p = null` since deleting forces a dictionary mode on plain JS objects . I'm not sure if that's true for DOM nodes but I'm still scared of `delete`.